Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -128,6 +127,13 @@ class FakespotViewController: UIViewController, Themeable, Notifiable, UIAdaptiv
         viewModel.isSwiping = false
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        guard #available(iOS 15.0, *) else { return }
```

Comment:
We should record telemetry even if we are not on iOS 15. We should just not send the extras in this case.